Jenkinsfile_nightly

```diff
@@ -59,9 +68,45 @@ withNightlyPipeline(type, product, component) {
   before('PerformanceTest') {
     sh "echo 'Benchmark is about to start'"
     sh "./gradlew benchmark --info --rerun-tasks && ./gradlew jmhReport --info --rerun-tasks"
+    print "calling loadPerformanceTestData gatling"
+    withSubscription(subscription) {
+      withTeamSecrets(config, params.ENVIRONMENT) {
+        withDocker('jbergknoff/postgresql-client', "--entrypoint='' -e PGPASSWORD=${DATABASE_PASS} -v ${WORKSPACE}/nightlyperformancedata/:/nightlyperformancedata") {
+          sh "chmod +x /nightlyperformancedata/load-or-delete-performance-testdata.sh"
+          sh "/nightlyperformancedata/load-or-delete-performance-testdata.sh \
+                        ${WORKSPACE} \
+                        'load' \
+                        ${DATABASE_USER} \
+                        ${DATABASE_NAME} \
+                        ${DATABASE_HOST} \
+                        ${DATABASE_PASS} \
+                        ${DATABASE_PORT} \
+                        ${params.ENVIRONMENT}"
+        }
+      }
+    }
+    print "done loadPerformanceTestData gatling"
   }
 
   after('PerformanceTest') {
+    print "calling deletePerformanceTestData gatling"
+    withSubscription(subscription) {
+      withTeamSecrets(config, params.ENVIRONMENT) {
+        withDocker('jbergknoff/postgresql-client', "--entrypoint='' -e PGPASSWORD=${DATABASE_PASS} -v ${WORKSPACE}/nightlyperformancedata/:/nightlyperformancedata") {
+          sh "chmod +x /nightlyperformancedata/load-or-delete-performance-testdata.sh"
+          sh "/nightlyperformancedata/load-or-delete-performance-testdata.sh \
+                        ${WORKSPACE} \
+                        'delete' \
+                        ${DATABASE_USER} \
+                        ${DATABASE_NAME} \
+                        ${DATABASE_HOST} \
+                        ${DATABASE_PASS} \
+                        ${DATABASE_PORT} \
+                        ${params.ENVIRONMENT}"
+        }
```

Comment:
as this could be used to set up data for other types of tests like functional tests as well, we should try to create a function and pass parameter like test type and script to be executed and try to avoid writing same code multiple times

Reply:
done